### ISSUE TITLE
NPE while create a new file

### ISSUE THREAD

**Issue by Chris2011** [Everlaw/nbts#52]
I use the newest npts version 1.8.10.0 and when add a new File via selecting the project, right click and go to new -> Typescript file, I got an NPE after creating and automatically opening the file.

```java
java.lang.NullPointerException
	at netbeanstypescript.TSService.updateFile(TSService.java:412)
	at netbeanstypescript.TSParser.parse(TSParser.java:60)
	at org.netbeans.modules.parsing.impl.TaskProcessor.callParse(TaskProcessor.java:621)
	at org.netbeans.modules.parsing.impl.SourceCache.getResult(SourceCache.java:251)
	at org.netbeans.modules.parsing.api.ResultIterator.getParserResult(ResultIterator.java:138)
	at org.netbeans.modules.csl.core.GsfReformatTask$1.run(GsfReformatTask.java:102)
	at org.netbeans.modules.parsing.impl.TaskProcessor.callUserTask(TaskProcessor.java:609)
	at org.netbeans.modules.parsing.api.ParserManager$UserTaskAction.run(ParserManager.java:153)
	at org.netbeans.modules.parsing.api.ParserManager$UserTaskAction.run(ParserManager.java:137)
	at org.netbeans.modules.parsing.impl.TaskProcessor$2.call(TaskProcessor.java:204)
	at org.netbeans.modules.parsing.impl.TaskProcessor$2.call(TaskProcessor.java:201)
	at org.netbeans.modules.masterfs.filebasedfs.utils.FileChangedManager.priorityIO(FileChangedManager.java:176)
	at org.netbeans.modules.masterfs.providers.ProvidedExtensions.priorityIO(ProvidedExtensions.java:360)
	at org.netbeans.modules.parsing.nb.DataObjectEnvFactory.runPriorityIO(DataObjectEnvFactory.java:141)
	at org.netbeans.modules.parsing.impl.Utilities.runPriorityIO(Utilities.java:88)
	at org.netbeans.modules.parsing.impl.TaskProcessor.runUserTask(TaskProcessor.java:201)
Caused: org.netbeans.modules.parsing.spi.ParseException
	at org.netbeans.modules.parsing.impl.TaskProcessor.runUserTask(TaskProcessor.java:209)
	at org.netbeans.modules.parsing.api.ParserManager.parse(ParserManager.java:104)
[catch] at org.netbeans.modules.csl.core.GsfReformatTask.reformat(GsfReformatTask.java:95)
	at org.netbeans.modules.editor.indent.TaskHandler$MimeItem.runTask(TaskHandler.java:557)
	at org.netbeans.modules.editor.indent.TaskHandler.runTasks(TaskHandler.java:316)
	at org.netbeans.modules.editor.indent.IndentImpl.reformat(IndentImpl.java:365)
	at org.netbeans.modules.editor.indent.api.Reformat.reformat(Reformat.java:154)
	at org.netbeans.modules.editor.indent.IndentScriptEngineHack.eval(IndentScriptEngineHack.java:96)
	at javax.script.AbstractScriptEngine.eval(AbstractScriptEngine.java:264)
	at org.netbeans.modules.editor.indent.IndentScriptEngineHack.eval(IndentScriptEngineHack.java:125)
	at javax.script.AbstractScriptEngine.eval(AbstractScriptEngine.java:249)
	at org.netbeans.modules.templates.ScriptingCreateFromTemplateHandler.createFromTemplate(ScriptingCreateFromTemplateHandler.java:133)
	at org.netbeans.api.templates.CreateFromTemplateImpl.build(CreateFromTemplateImpl.java:121)
	at org.netbeans.api.templates.CreateFromTemplateImpl.build(CreateFromTemplateImpl.java:92)
	at org.netbeans.api.templates.FileBuilder.build(FileBuilder.java:250)
	at org.netbeans.api.templates.FileBuilder.createFromTemplate(FileBuilder.java:351)
	at org.openide.loaders.MultiDataObject.handleCreateFromTemplate(MultiDataObject.java:878)
	at org.netbeans.modules.csl.core.GsfDataObject.handleCreateFromTemplate(GsfDataObject.java:160)
	at org.openide.loaders.DataObject$CreateAction.run(DataObject.java:1597)
	at org.openide.loaders.DataObjectPool$1WrapAtomicAction.run(DataObjectPool.java:261)
	at org.openide.filesystems.EventControl.runAtomicAction(EventControl.java:127)
	at org.openide.filesystems.FileSystem.runAtomicAction(FileSystem.java:519)
	at org.openide.loaders.DataObjectPool.runAtomicAction(DataObjectPool.java:286)
	at org.openide.loaders.DataObject.invokeAtomicAction(DataObject.java:1051)
	at org.openide.loaders.DataObject.createFromTemplate(DataObject.java:983)
	at org.openide.loaders.TemplateWizard$DefaultIterator.instantiate(TemplateWizard.java:919)
	at org.openide.loaders.TemplateWizard.handleInstantiate(TemplateWizard.java:605)
	at org.openide.loaders.TemplateWizard.instantiateNewObjects(TemplateWizard.java:439)
	at org.openide.loaders.TemplateWizardIterImpl.instantiate(TemplateWizardIterImpl.java:248)
	at org.openide.loaders.TemplateWizardIteratorWrapper.instantiate(TemplateWizardIteratorWrapper.java:160)
	at org.openide.WizardDescriptor.callInstantiateOpen(WizardDescriptor.java:1629)
	at org.openide.WizardDescriptor.callInstantiate(WizardDescriptor.java:1570)
	at org.openide.WizardDescriptor.access$2300(WizardDescriptor.java:92)
	at org.openide.WizardDescriptor$Listener$2$1.run(WizardDescriptor.java:2257)
	at org.openide.util.RequestProcessor$Task.run(RequestProcessor.java:1443)
	at org.netbeans.modules.openide.util.GlobalLookup.execute(GlobalLookup.java:68)
	at org.openide.util.lookup.Lookups.executeWith(Lookups.java:303)
	at org.openide.util.RequestProcessor$Processor.run(RequestProcessor.java:2058)
```

**Comment by pmarquez** [comment]
I am getting the exact same error on the same version of the plugin as does Chris2011. (Just so you know  ;)  ).

**Comment by jeffrey-easyesi** [comment]
Fixed in release 1.8.10.2
